homepage.css

```diff
@@ -20,55 +24,57 @@ body{
 
 nav{
 	width:100%;
-	background:white;
+	background-color:#0C59CF;
```

Comment:
26/8/2021: Change the background color of the nav to navy blue which was my color theme that I decided to apply.